addon/components/imgix-image.js

```diff
@@ -206,18 +221,33 @@ export default Component.extend({
         } else {
           let showARWrongFormatWarning = false;
           const buildSrcSetPair = targetWidth => {
+            const targetHeight = (() => {
+              if (options.h) {
+                return options.h;
+              }
+
+              const aspectRatioDecimal = parseAspectRatio(aspectRatio);
+              if (aspectRatio != null && aspectRatioDecimal === false) {
+                // false indicates invalid
+                showARWrongFormatWarning = true;
+              }
+              if (aspectRatioDecimal && aspectRatioDecimal > 0) {
+                return Math.ceil(targetWidth / aspectRatioDecimal);
+              }
+
+              return options.h;
```

Comment:
This is always going to return a crap value, right? We should just remove this return.

Reply:
This is fixed in a subsequent PR 😌